be/src/util/doris_metrics.cpp

```diff
@@ -197,6 +230,37 @@ DorisMetrics::DorisMetrics() : _metric_registry(_s_registry_name) {
     _server_metric_entity = _metric_registry.register_entity("server");
 
     INT_COUNTER_METRIC_REGISTER(_server_metric_entity, fragment_requests_total);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, transmit_data);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, transmit_data_by_http);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, exec_plan_fragment);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, exec_plan_fragment_prepare);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, exec_plan_fragment_start);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, cancel_plan_fragment);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, fetch_data);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, fetch_table_schema);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, tablet_writer_open);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, tablet_writer_add_block);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, tablet_writer_add_block_by_http);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, tablet_writer_cancel);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, get_info);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, update_cache);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, fetch_cache);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, clear_cache);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, merge_filter);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, apply_filter);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, transmit_block);
+    INT_COUNTER_METRIC_REGISTER(_server_metric_entity, transmit_block_by_http);
```

Comment:
actually, there are these metrics in bvar in brpc already.

Reply:
Provide two metrics interfaces for the same node
- http://ip:http_port/metrics
- http://ip:brpc_port/brpc_metrics

It is not very reasonable from the perspective of use. I think it is reasonable and acceptable to have redundant data here, and it does not affect performance